CODEOWNERS

```diff
@@ -0,0 +1,39 @@
+## Dialects, Conversions, Tools
+
+# Arc
+**/Dialect/Arc @fabianschuiki
+tools/arcilator @fabianschuiki
+
+# ESI
+**/Dialect/ESI @teqdruid
+
+# FIRRTL
+**/FIRRTL* @darthscsi @dtzSiFive @seldridge
+*/firtool @dtzSiFive
+libs/Firtool @dtzSiFive
+
+# LLHD
```

Comment:
Maybe throw @maerhart under LLHD and Arc?

Reply:
Sure, that's fine by me.

Reply:
Yes, updated!